mix/sphinx.nim

```diff
@@ -114,33 +116,25 @@ proc generateRandomDelay(): seq[byte] =
 const paddingLength = (((t + 1) * (r - L)) + 2) * k
 
 # Function to compute betas, gammas, and deltas
-proc computeBetaGammaDelta(
-    s: seq[seq[byte]],
-    hop: openArray[Hop],
-    msg: Message,
-    delay: openArray[seq[byte]],
-    destHop: Hop,
-): Result[(seq[byte], seq[byte], seq[byte]), string] = # TODO: name tuples
+proc computeBetaGamma(
```

Comment:
NIT

don't forget to update comment.

also, the comment for this function feels a bit like:

```C
// set i to zero
int i = 0
// while the iteration counter is less than 10
while (i < 10) {
  //...
  
  // increment the iteration counter
  i += 1;
}
```

Perhaps something that contextualizes the computation.